scripts/schemas/twister/testsuite-schema.yaml

```diff
@@ -156,6 +156,9 @@ mapping:
         required: false
         sequence:
           - type: str
+      "simulation_exclude":
+        type: any
```

Comment:
I would suggest to use seq instead of any, any is becuase we want backward compatible with string and list.

Reply:
Thanks for the suggestion, I've updated it to use seq + enum